spec/unit/lib/temporal/testing/local_workflow_context_spec.rb

```diff
@@ -126,4 +126,23 @@ def execute
     # Heartbeat doesn't do anything in local mode, but at least it can be called.
     workflow_context.execute_activity!(TestHeartbeatingActivity)
   end
+
+  it 'can await with false condition' do
```

Comment:
nit: this needs to be in `describe '#await' do` block for clarify and to match our current testing structure